nsqadmin/static/js/views/channel.hbs

```diff
@@ -176,8 +176,12 @@
             <tr>
                 <th>Client Host</th>
                 <th>User-Agent</th>
+                <th>Client Region</th>
+                <th>Client Zone</th>
                 <th>Attributes</th>
```

Comment:
style/ux comments:

In general i feel this makes the table too busy.

I think these would be better added to the Attributes field as a label (w/ a specific coloring for the zone + region value). I think the host zone/region should similarly be labels in the `nsqd host` cell.

The think you should be able to get at a glance is when a connection is a zone-local match, or when a connection is a region-local match. I would almost be ok not showing the nsqd host zone/region if we added that. One option -we could use a coloring of the background on the row to flag zone and region local? should we add region + zone in grey but color them when they are a zone-local or region-local match so they pop?

Reply:
haha, we also originally though it would be good to do a colored matching, i will look back into this

Reply:
also to clarify the framing of my comments - the existing state is fine to land as a starting point.